src/status_im/keycard/core.cljs

```diff
@@ -542,3 +562,41 @@
       :keycard
       :from-key-storage-and-migration?
       boolean))
+
+(fx/defn start-nfc
+  {:events [:keycard.ui/start-nfc]}
+  [cofx]
+  {:keycard/start-nfc nil})
+
+(fx/defn stop-nfc
+  {:events [:keycard.ui/stop-nfc]}
+  [cofx]
+  {:keycard/stop-nfc nil
+   :keycard.callback/on-card-disconnected nil})
+
+(fx/defn start-nfc-success
+  {:events [:keycard.callback/start-nfc-success]}
+  [{:keys [db]} _]
+  (log/debug "[keycard] nfc started success")
+  {:db (assoc-in db [:keycard :nfc-running?] true)})
+
+(fx/defn start-nfc-failure
+  {:events [:keycard.callback/start-nfc-failure]}
+  [{:keys [db]} _]
+  (log/debug "[keycard] nfc failed starting")
+  {}) ;; leave current value on :nfc-running
```

Comment:
`log/debug` should return nil, so that should just work without explicitly returning `{}`